packages/admin/src/pages/super-admin-dashboard/user/[id]/change-department.page.tsx

```diff
@@ -78,7 +78,7 @@ const UserPage = ({
               value: department.id,
             }))}
             fieldErrors={fieldErrors}
-            defaultChecked={user.department?.name}
+            defaultChecked={user.department?.name ?? departments[0].name}
```

Comment:
If the user doesn't have a department assigned then I don't think we want any default value to be checked.  Think you can assign to false in this case